.gitignore

```diff
@@ -43,6 +43,8 @@ pip-log.txt
 nosetests.xml
 
 # generated examples
-/examples/boards/ppro/xilinx
+/examples/boards/papilio_pro/xilinx
```

Comment:
There should be a better way to do this, otherwise we are at risk that this ignore file could contain too many exceptions.  Maybe something like `/examples/*/xilinx` would be more generic?